.editorconfig

```diff
@@ -0,0 +1,33 @@
+# https://editorconfig.org
+# Editor Config for WebMemex
+# Download an extension for editor config to allow your IDE
+# to follow the rules of the project
+
+# Used to indicate the file is in root of the project
+root = true
+
+# Styling applicable to all forms of files
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+end_of_line = lf
+charset = utf-8
+
+# Styling applicable to the file extension given
+[*.js, *.jsx]
+# This is due to the fact that github allows only 119 columns of space to view
+max_line_length = 119
```

Comment:
The reason here is actually 120, but minus 1 char for the diff view. Why 120 I don't get because it seems pretty much browser specific, and people just choose a round number, see:
![screenshot of github editor in different browsers on OsX](https://lh3.googleusercontent.com/-dyEc07nEwco/VAifbvQ8LEI/AAAAAAAAAM4/W15JYi5AZ4Y/s1600/screenshot.png)
(source [Django developers discussion](https://groups.google.com/forum/#!msg/django-developers/3o7i18ewMWs/JaCaf7Cjef0J))
For more random numbers, see [this stackoverflow thread](http://stackoverflow.com/q/22207920).

I do agree setting line lengths to something higher than 80, but 119 and binding that to Github web view and some browser specifics and then rounding that number seems pretty ... arbitrary.